rskj-core/src/main/java/co/rsk/core/bc/BlockChainImpl.java

```diff
@@ -191,9 +191,9 @@ public ImportResult tryToConnect(Block block) {
                 logger.info("block: num: [{}] hash: [{}], processed after: [{}]nano, result {}", block.getNumber(), block.getShortHash(), totalTime, result);
                 return result;
             }
-        } catch (Throwable th) {
-            logger.error("Unexpected error: ", th);
-            panicProcessor.panic("bcerror", th.toString());
+        } catch (Exception e) {
```

Comment:
.

Reply:
It will remain being a Throwable, until better check